src/store/getters.ts

```diff
@@ -7,24 +7,28 @@ import PropertyValueRelation from '@/data-model/PropertyValueRelation';
 export default {
 	query( rootState: RootState ): QueryRepresentation {
 		return {
-			condition: {
-				propertyId: rootState.conditionRow.propertyData.id,
-				value: rootState.conditionRow.valueData.value,
-				propertyValueRelation: rootState.conditionRow.propertyValueRelationData.value,
-			},
+			conditions: [ {
```

Comment:
I think for this PR it is enough to still just return a single condition here. While we have to extend QueryRepresentation to take multiple conditions at some point, doing so here makes this change to big.